bin/resources/GameIndex.yaml

```diff
@@ -23158,7 +23194,13 @@ SLPM-62035:
   name: "Got to Do! Hot Spring Table Tennis"
   region: "NTSC-J"
 SLPM-62036:
-  name: "Choukousoku Reversi"
+  name: "Chou Kousoku Reversi"
```

Comment:
Are you sure on this? The internet is pretty unanimous on "Choukousoku" being a single word (same goes for the next 2 entries also.